api/team.go

```diff
@@ -489,3 +492,70 @@ func getMyTeam(c *Context, w http.ResponseWriter, r *http.Request) {
 		return
 	}
 }
+
+func importTeam(c *Context, w http.ResponseWriter, r *http.Request) {
+	if !c.HasPermissionsToTeam(c.Session.TeamId, "import") || !c.IsTeamAdmin(c.Session.TeamId) {
+		c.Err = model.NewAppError("importTeam", "Only a team admin can import data.", "userId="+c.Session.UserId)
+		c.Err.StatusCode = http.StatusForbidden
+		return
+	}
+
+	if err := r.ParseMultipartForm(10000000); err != nil {
```

Comment:
I intentionally left that out for now. There is going to be another pass of this code after the admin Console stuff gets implemented where many more things about the import process are going to be made configurable. This will be fixed then, after admin console stuff is ready.

For now, I figured I would leave it out so there is one less thing for the person doing the admin console to fix.